jest.config.js

```diff
@@ -16,15 +16,16 @@ const moduleNameMapper = getPackageNames().reduce(
       '<rootDir>/packages/react-html/src/server/index.ts',
     '@shopify/react-network/server':
       '<rootDir>/packages/react-network/src/server.ts',
+    '@shopify/address-consts': '<rootDir>/packages/address-consts/src/index.ts',
```

Comment:
This one drove me up the wall. `@shopify/address-conts` was not getting _module mapped_ properly. 
`require.resolve('@shopify/address-consts')`  was resolving to `quilt/packages/address/src/index.ts`, instead of `quilt/packages/address-consts/src/index.ts`. Moving this up in the list manually got the module resolving correctly.

Reply:
I think we could also fix this by adding a `$` to the `@shopify/${name}` bit, presumably that part is a regex and `address` matches `address-consts`